src/ConnectorRuntimeConfig.ts

```diff
@@ -3,6 +3,8 @@ import * as log4js from "log4js";
 import { ConnectorRuntimeModuleConfiguration } from "./ConnectorRuntimeModule";
 
 export interface ConnectorRuntimeConfig extends RuntimeConfig {
+    mode: "debug" | "production";
```

Comment:
Wäre hier nicht eher ein ConnectorMode.Debug | ConnectorMode.Production?

Reply:
Ne, in der config sind es ja noch strings.